src/utils/self_test/SConscript

```diff
@@ -13,19 +13,18 @@ SELF_TEST = 'self_test.c'
 def scons():
     """scons function"""
 
-    Import('env', 'prereqs', 'cart_lib', 'gurt_lib')
-
+    Import('env', 'prereqs', 'cart_lib', 'gurt_lib', 'cart_utils_lib')
     tenv = env.Clone()
 
     prereqs.require(tenv, 'pmdk')
     tenv.AppendUnique(CPPPATH=['#/src/cart', '#/src/tests/ftest/cart'])
-    tenv.AppendUnique(LIBS=['cart', 'gurt', 'pthread', 'm'])
+    tenv.AppendUnique(LIBS=['cart', 'gurt', 'pthread', 'm', 'cart_utils'])
     prereqs.require(tenv, 'mercury')
-    tenv.AppendUnique(LIBPATH=['#/src/cart'])
+    tenv.AppendUnique(LIBPATH=[Dir('../../cart')])
```

Comment:
This should not be needed since cart adds itself to default environment in src/cart/SConscript


env.AppendUnique(LIBPATH=[Dir('.')])

Reply:
Tested and agreed. Remove.